src/content/self-hosted/install/releases.mdx

```diff
@@ -8,11 +8,41 @@ id: releases
 ## 1.7.0
 TBD
 
+### Removal Notice
+- Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) is no longer supported.
+
 ### Features
 
 - Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
 - Support for configuring the [service account](self-hosted/administration/configuration.mdx#buildkit) of buildkit.
 - Support for [token-based authentication](self-hosted/administration/configuration.mdx#token) to simplify the installation of Okteto Self-Hosted.
+- [Personal Access Tokens](cloud/personal-access-tokens.mdx) now have expiration date. They expire after 180 days of their creation. For existing tokens, the expiration date is set to 180 days from when your instance is upgraded to this version
+- Update the Okteto CLI version to [2.14.1](https://github.com/okteto/okteto/releases/tag/2.14.1)
+- Allow to delete external resources individually from the UI
+- Default installation of Okteto doesn't require an external storage for the registry configured. It uses filesystem storage by default
+- Default installation of Okteto doesn't require a PVC for [BuildKit](self-hosted/administration/configuration.mdx#token). It uses [filesystem storage](self-hosted/administration/configuration.mdx#filesystem) by default
```

Comment:
Is this a breaking change? I had to make changes on my values.yaml when going from 1.6 to 1.7RC to keep it using volumes

Reply:
Do we have docs on why you should use pvc vs file system?

Reply:
> Is this a breaking change? I had to make changes on my values.yaml when going from 1.6 to 1.7RC to keep it using volumes 

I tested 1.7-rc.6 and the fix was working, wat rc was this happening on? I updated yesterday from 1.6 with PVC and then with the same config file to 1.7-rc6 and PVC was still being used

Reply:
@rberrelleza no changes are needed. You had to do changes because you hit a bug in the "migration" logic to keep using previous configuration. That bug was detected and fixed in the current release candidate. So, it shouldn't be a breaking change, we detect old configuration and we keep working as it was before.


> Do we have docs on why you should use pvc vs file system?

As far as I know, no, we don't have such documentation. I know Pablo had in mind to update some documentations about these new default values, but I don't see something like that